guide/getting-started/hardware.md

```diff
@@ -68,13 +69,14 @@ You may already be familiar with key fobs from your bank or work. They are small
    height = 400
 %}
 
-Hardware wallets for bitcoin play a similar role but with several differences. Like key fobs, they securely store an authentication mechanism and isolate it from other devices and networks. In bitcoin, these are the seed, associated keys, and all functions that require direct interaction with them.
+Bitcoin hardware wallets play a similar role but with several noteworthy differences. Like key fobs, they securely store an authentication method and isolate it from the Internet, networks, and other devices. In bitcoin, these are the seed, associated keys, and functions that require interaction with them.
 
-Because hardware wallets are so optimized for this specific use case, they have tiny screens and limited input options. Most interactions with them happen via desktop software that the manufacturer also provides. Users perform most wallet interactions on their desktops, and the application sends only specific requests to the hardware wallet to verify or sign data. This provides a good balance of security and convenience and keeps hardware wallets generally affordable for most use cases.
+Because hardware wallets are so optimized for this specific use case, they have tiny screens and limited input options. Most interactions with them happen via desktop software that the manufacturer provides. The application sends only specific requests to the hardware wallet to verify or sign data. This provides a strong balance of security and convenience and keeps hardware wallets generally affordable.
 
 ## Nodes
 
-Thousands of computers participate in the Bitcoin network. They are called nodes and continuously exchange, verify and compare transaction and block data. Running your own node, instead of relying on a third-party node, has the benefit that transactions are getting validated by your own personal device. Nodes also improve privacy and help secure the network. Several DIY and consumer hardware products have been created to make it easy to run your own node on a dedicated device.
+Thousands of computers participate in the Bitcoin network. They are called nodes. Nodes exchange, verify, and compare block and transaction data.
+Running your own node, instead of relying on a third-party node, lets you validate transactions on your own device. Nodes also improve privacy and help secure the network. Several DIY and consumer hardware products make it easy to run your own node.
```

Comment:
```suggestion
Running your own node, instead of relying on a third-party node, lets you validate transactions on your own device. It also improves the privacy of your transactions and help secure the network. Several DIY and consumer hardware products make it easy to run your own node.
```

Reply:
We should add a link to the [Glossary node page](https://bitcoin.design/guide/glossary/node/) in a separate PR.

Reply:
Added https://github.com/BitcoinDesign/Guide/pull/438/commits/6a36ce616e679c1b4925d434411b8d3d550b29fd